src/software/ai/hl/stp/tactic/move/move_tactic.h

```diff
@@ -34,12 +34,16 @@ class MoveTactic : public Tactic
      * @param dribbler_mode The dribbler mode
      * @param ball_collision_type how to navigate around the ball
      * @param auto_chip_or_kick The command to autochip or autokick
+     * @param max_allowed_speed_mode The mode of maximum speed allowed
+     * @param target_spin_rps The target spin while moving in revolutions per second
      */
     void updateControlParams(
         Point destination, Angle final_orientation, double final_speed,
-        DribblerMode dribbler_mode            = DribblerMode::OFF,
-        BallCollisionType ball_collision_type = BallCollisionType::AVOID,
-        AutoChipOrKick auto_chip_or_kick      = {AutoChipOrKickMode::OFF, 0});
+        DribblerMode dribbler_mode                 = DribblerMode::OFF,
+        BallCollisionType ball_collision_type      = BallCollisionType::AVOID,
+        AutoChipOrKick auto_chip_or_kick           = {AutoChipOrKickMode::OFF, 0},
+        MaxAllowedSpeedMode max_allowed_speed_mode = MaxAllowedSpeedMode::PHYSICAL_LIMIT,
+        double target_spin_rps                     = 0.0);
```

Comment:
similar comment here about using `radians` instead of `revolutions` and spelling out the units explicitly